src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.spec.ts

```diff
@@ -142,15 +142,15 @@ describe('ApiInterceptorService', () => {
         {
           status: 0,
           statusText: 'Unknown Error',
-          message: 'Http failure response for (unknown url): 0 Unknown Error',
+          message: 'HTTP failure response for (unknown url): 0 Unknown Error',
```

Comment:
@anthonyeleven : these are actually unit tests testing the response we get from the angular http package. Which means that these particular messages are not set from the dashboard code but from an external package. I just searched for that particular package and bumped [into this line](https://github.com/angular/angular/blob/main/packages/common/http/src/response.ts#L364), where the message is being set. So this changes will fail our [unit tests as you see here](https://jenkins.ceph.com/job/ceph-pull-requests/154537/testReport/junit/projectroot.src.test/mgr/mgr_dashboard_frontend_unittests/). I don't know why they chose to follow this standard, but maybe they were going for the PascalCase which is usually followed across typescript.